examples/pytorch/question-answering/recipes/uni_80blocksparse_freq0.01_18prune10fine.yaml

```diff
@@ -0,0 +1,871 @@
+version: 1.1.0
```

Comment:
Need to make this a markdown file and include details on how to run it such as the command

Reply:
Can we also simplify this recipe so all similar layers are grouped together? Additionally can we create reference variables at the top like we do for other recipes such as yolov5?

Reply:
will update with recipes in sparseml